pylint/checkers/refactoring.py

```diff
@@ -27,6 +27,52 @@ def _all_elements_are_true(gen):
     return values and all(values)
 
 
+def _get_exception_handlers(node, exception):
```

Comment:
`pylint.checkers.utils.node_ignores_exception` is doing something every similar to this, except that it's returning a boolean instead of the list of handlers. So that we're doing handler detection consistently, please could you split `utils.node_ignores_exception` into a `utils.get_exception_handlers` function, and then `utils.node_ignores_exception` can call this and return True if there are any exception handlers.

Reply:
Thanks @AWhetter  for this review. In fact i get inspired from ``pylint.checkers.utils.node_ignores_exception``. I will do what you are asking asap.